### PR TITLE
Adds chat font scale option

### LINKED ISSUE #7928: chat text is way too small
### Environment
- Game Version: Staging 21120412
- OS Version: Arch Linux

### Detailed description of bug

Chat log text has become really small, and at least for me on my 1366x768 screen is increibly hard to read.

here a comparison with the work in progress message
text

### PR DESCRIPTION
**Pull request by MaxIsJoe**
closes #7928

CL: [New] Added the option to control font sizes for chat. This is an important accessibility feature for players who have trouble reading.

### CODE REVIEW

File: UnityProject/Assets/Scripts/Core/Chat/Chat.Process.cs [hunk 1]
```diff
@@ -364,7 +364,7 @@ private static (string, ChatModifier) ProcessMessage(PlayerInfo sentByPlayer, st
 		return AddMsgColor(channels,
 			$"{chan}<b>{speaker}</b> {verb}" // [cmd]  Username says,
 			+ "  " // Two hair spaces. This triggers Text-to-Speech.
-			+ $"<size={textSize}>" + "\"" +  message + "\"" + "</size>"); // "This text will be spoken by TTS!"
+			+ $"<size=+{textSize}>" + "\"" +  message + "\"" + "</size>"); // "This text will be spoken by TTS!"
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/Core/Chat/Chat.Process.cs:367]: what does the + sign do here?

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/Core/Chat/Chat.Process.cs:367]: the + sign in TMPro adds into the size rather than statically defining a hard size

Comment (corp-0) [reply UnityProject/Assets/Scripts/Core/Chat/Chat.Process.cs:367]: does it look exactly the same with the default font size setting?

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/Core/Chat/Chat.Process.cs:367]: yes, it doesn't follow the font scale so you need to add.

**Review by corp-0 — APPROVED** [review]

**Pull request merged**